.travis.yml

```
@@ -8,7 +8,7 @@ rvm:
 - 2.5
 - 2.4
 - 2.3
-- jruby-9k
+- jruby-head
```

I think we should use the latest stable release:

```suggestion
- jruby-9.2.9.0
```
`jruby-head` resolves to `9.2.10.0-SNAPSHOT` and snapshots, by definition, are unstable.

---

Agree. We used jruby-head in the early days before 9.x was available. But that situation has long since passed.

---

Alright, but this means tracking and understanding these upstream releases which I would rather not do... Can't they have a pointer to `$latest_stable`?

I'm going to go with 9.2.9.0 but I foresee that we will forget to keep it current.

---

I think RVM understands 9.2, which would cover you for quite a long time. Generally, I recommend following what Asciidoctor core does, so you won't have to track it outside of our own ecosystem.